dts/bindings/pinctrl/cypress,psoc6-pinctrl.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2020, Linaro Limited
+# Copyright (c) 2020, ATL Electronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: Cypress PSoC-6 Pinctrl container node
+
+compatible: "cypress,psoc6-pinctrl"
+
+include: [base.yaml]
+
+properties:
+    "#address-cells":
+      required: true
+      const: 1
+    "#size-cells":
+      required: true
+      const: 1
+
+child-binding:
+    description: |
+      Cypress PSoC-6 pins.  This flags are already present on the device tree
+      for the pins that a particular peripheral uses.  A pin that doesn't
+      define flags is selected for analog functions.
+    properties:
+      "cypress,pins":
+        type: phandle-array
+
+      bias-pull-up:
```

Comment:
Can you see if you can pull the common properties here out into a pinctrl-flags.yaml and see if we can include that both here and in `atmel,sam-pinctrl.yaml`

Reply:
Done at 05ba19e84abd1830a78ab60ed519a63711d5744a